Directory.Build.props

```diff
@@ -56,6 +56,12 @@
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(IsPackable)' == 'true'">
```

Comment:
after a IdWeb release, we need to update the `PackageValidationBaselineVersion`

Reply:
can you explain the `Condition="'$(IsPackable)' == 'true'"` condition?

Reply:
the pack step is failing in package baseline validation. The errors that read “Type ‘…’ exists on [Baseline] lib/net7.0 … but not on lib/netstandard2.0” are from the validator comparing our last shipped net7.0 assets to the current package after we dropped net7.0. Those “missing on current TFMs” are expected once 7.0 is removed; they should be ignored for baseline checks.